libs/arangodb/langchain_arangodb/chains/graph_qa/arangodb.py

```diff
@@ -283,9 +524,11 @@ def _call(
                     """
                     raise ValueError(error_msg)
 
-            _run_manager.on_text(
-                f"AQL Query ({aql_generation_attempt}):", verbose=self.verbose
-            )
+            query_message = f"AQL Query ({aql_generation_attempt})"
+            if cached_query:
+                query_message += f" (used cached query, score: {score})"
+
+            _run_manager.on_text(query_message, verbose=self.verbose)
```

Comment:
I think we need to add a newline before printing the AQL query

```suggestion
            _run_manager.on_text(f"{query_message}\n", verbose=self.verbose)
```

Reply:
this has been changed, see [2ccc061](https://github.com/arangoml/langchain-arangodb/pull/15/commits/2ccc06147ddff23369382e61074f59f6d1cd87de)